src/ECharts/Series.purs

```diff
@@ -0,0 +1,699 @@
+module ECharts.Series (
+  setSeries,
+  Series(..),
+  universalSeriesDefault,
+  lineSeriesDefault,
+  barSeriesDefault,
+  scatterSeriesDefault,
+  pieSeriesDefault,
+  radarSeriesDefault,
+  chordSeriesDefault,
+  forceSeriesDefault,
+  candlestickSeriesDefault,
+  mapSeriesDefault,
+  gaugeSeriesDefault,
+  funnelSeriesDefault,
+  eventRiverSeriesDefault
+  ) where 
+
+
+import Control.Monad.Eff
+import Data.Function
+import Data.Maybe
+import Data.Argonaut.Core
+import Data.Argonaut.Encode
+import Data.Argonaut.Combinators
+import Data.Tuple
+import Data.StrMap
+import Data.Array (concat)
+
+import ECharts.Common
+import ECharts.Coords
+import ECharts.Chart
+import ECharts.Tooltip
+import ECharts.Type
+import ECharts.Style.Item
+import ECharts.Mark.Line
+import ECharts.Mark.Point
+import ECharts.Item.Data
+import ECharts.Symbol
+import ECharts.Series.Force
+import ECharts.Series.Gauge
+import ECharts.Series.EventRiver
+import ECharts.Axis
+import ECharts.Title
+import ECharts.Utils
+
+data Series = LineSeries
+              {common :: UniversalSeriesRec, special :: LineSeriesRec}
```

Comment:
I would prefer to the `special` fields with more descriptive names, such as, `lineSeries`, `barSeries`, etc. These names also match the names of the defaults and the record types.